src/olympia/files/models.py

```diff
@@ -203,19 +202,10 @@ def generate_filename(self):
         file_extension = '.xpi' if self.is_signed else '.zip'
         return '-'.join(parts) + file_extension
 
-    _pretty_filename = re.compile(r'(?P<slug>[a-z0-7_]+)(?P<suffix>.*)')
-
-    def pretty_filename(self, maxlen=20):
-        """Displayable filename.
-
-        Truncates filename so that the slug part fits maxlen.
-        """
-        m = self._pretty_filename.match(self.filename)
-        if not m:
-            return self.filename
-        if len(m.group('slug')) < maxlen:
-            return self.filename
-        return '{}...{}'.format(m.group('slug')[0 : (maxlen - 3)], m.group('suffix'))
+    @property
+    def pretty_filename(self):
+        """Displayable filename."""
+        return os.path.basename(self.filename) if self.filename else ''
```

Comment:
It's quite a lot less functional than the existing `pretty_filename` - it doesn't sanitize or trim to a max length.  I guess that's okay though? (we do the sanitizing earlier and wrap in css?)

Reply:
Yeah. Ultimately it was only used in a single place - devhub - everywhere else was already using the raw filename...